### PR TITLE
Autopilot Sync Environment Content

### LINKED ISSUE #7142: Autopilot Clone Feature
Re: <Add Doc title and/or link>

Priority: Low/Medium/High (choose one, remove the other options)

## Issue Description

* Add new content on the ability to clone an environment before running Autopilot. This is part of [Otto issue to Update Public Docs.
](https://getpantheon.atlassian.net/browse/OTTO-1786)
### How will this impact users?

*

### Context

* 

## Suggested Resolution

* 

## Additional Information

*

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes #7142 

## Summary

**[Autopilot](https://pantheon.io/docs/guides/autopilot/enable-autopilot/)** - Added new Sync Environment content.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7236](https://pr-7236--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `49` | `55.00000000000001` | `6` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7236` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/35594/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/35594/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7236--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by whitneymeredith** [comment]
[Preview](https://pr-7236--pantheon-docs.my.pantheonfrontend.website/docs/guides/autopilot/enable-autopilot/)

**Comment by wordsmither** [comment]
lgtm

File: source/content/guides/autopilot/02-enable-autopilot.md [hunk 1]
```diff
@@ -66,41 +66,45 @@ To work with Autopilot, [switch to the Workspace](/guides/new-dashboard/workspac
...
-- Weekly
-- Monthly
+    - Never (Update Manually)
+    - Weekly
+    - Monthly
 
-Click **Save** to save the changes.
+1. Select the **Sync Live Environment** checkbox to sync environments.
+
+   The **Sync Environment** feature syncs your Live environment to your Dev environment. Your databases and files will be synced, but not your code. If the Dev code does not match the Live code, Autopilot will use the code in the Dev environment.
```
Comment (kelly-mcgonnigle) [inline source/content/guides/autopilot/02-enable-autopilot.md:101]:
```suggestion
   The **Sync Environment** feature syncs your Live environment to your Dev environment before Autopilot performs updates. Your databases and files will be synced, but not your code. If the Dev code does not match the Live code, Autopilot will use the code in the Dev environment.
```

File: source/content/guides/autopilot/02-enable-autopilot.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ showtoc: true
 anchorid: enable-autopilot
 permalink: docs/guides/autopilot/enable-autopilot/
 editpath: autopilot/02-enable-autopilot.md
-reviewed: "2021-08-10"
+reviewed: "2022-04-05"
 ---
 
 ## Enable Autopilot
```
Comment (kelly-mcgonnigle) [inline source/content/guides/autopilot/02-enable-autopilot.md:16]: We're missing some structure in this section. Can we add headers for the 3 steps in the onboarding process (similar to how the "Autopilot Configuration - Manage Autopilot Settings" is laid out)? Also Environment Syncing is an option during onboarding, just as it is in the Configuration Settings. So we need steps and screenshots for that. Let me know if you have trouble finding this in the Figma file i sent you.

File: source/content/guides/autopilot/05-troubleshoot.md [hunk 3]
```diff
@@ -336,7 +336,7 @@ Unfreeze the site in the old dashboard before re-queueing updates. Note, unfreez
 
 ### Diagnosis
 
-Site is running a framework that is not Drupal, Drupal 8, or WordPress. Currently, WordPess Multisite (`wordpress_network`) is not supported.
+Site is running a framework that is not Drupal, Drupal 8, or WordPress. WordPess Multisite (`wordpress_network`) is not supported.
```
Comment (kelly-mcgonnigle) [inline source/content/guides/autopilot/05-troubleshoot.md:339]: Can we revert this change? We plan to support Multisite in the future so "currently" is accurate.

File: source/content/guides/autopilot/02-enable-autopilot.md (hunk 2 above)
Comment (whitneymeredith) [reply source/content/guides/autopilot/02-enable-autopilot.md:16]: @kelly-mcgonnigle Can you clarify which 3 steps you're referring to?

Comment (kelly-mcgonnigle) [reply source/content/guides/autopilot/02-enable-autopilot.md:16]: Yep! I explain in this Loom. let me know if you have follow up qs: https://www.loom.com/share/d8387d3e112540dca896b6e44689c8b8

File: source/content/guides/autopilot/05-troubleshoot.md (hunk 3 above)
Comment (whitneymeredith) [reply source/content/guides/autopilot/05-troubleshoot.md:339]: Done!

**Comment by whitneymeredith** [comment]
@kelly-mcgonnigle Thanks! I made changes based on your suggestions / requests. Check out the updated version [here](https://pr-7236--pantheon-docs.my.pantheonfrontend.website/docs/guides/autopilot/enable-autopilot/), and let me know what you think.

**Pull request merged**